package.json

```diff
@@ -43,9 +43,11 @@
     "@wireapp/react-ui-kit": "1.1.4",
     "babel-plugin-react-intl": "2.4.0",
     "babel-polyfill": "6.26.0",
+    "dexie-batch": "ffflorian/dexie-batch#native-promise",
```

Comment:
@ffflorian Let's send a PR to https://github.com/raphinesse/dexie-batch and add a SHA hash here...